Tahi-project/tahi
# Convert manuscript manager to react

Manuscript manager was using rails stuff to render. This translates it over to React.
# QA

Make sure you can open a card, post a new one, and on reload it is persisted.
_edit:_ you can no longer post and persist a card, the submit functionality was removed from the button in a previous pr
